chips/nrf52/src/ppi.rs

```diff
@@ -0,0 +1,90 @@
+use kernel::common::VolatileCell;
```

Comment:
This needs to be documented. What is the PPI? What is it good for? How do I use it?